src/quicklist.c

```diff
@@ -136,10 +149,40 @@ quicklist *quicklistNew(int fill, int compress) {
     return quicklist;
 }
 
+void quicklistRepr(unsigned char *ql, int full) {
```

Comment:
Should this method be moved to the end, as it's defined at the end of quicklist.h.